core/src/main/java/org/mule/runtime/core/api/routing/filter/FilteredException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.api.routing.filter;
+
+import org.mule.runtime.api.exception.MuleException;
+
+/**
+ * Indicates that the execution of the current event is stopped. This exception is thrown to indicate this condition to the source
+ * of the flow.
+ * 
+ * @since 4.0
+ */
+public class FilteredException extends MuleException {
```

Comment:
This should be ValidationException and the error type VALIDATION. We need to talk to MG so he start using that same type as parent for the validations of the validation module.